geoh5py/objects/object_base.py

```diff
@@ -72,8 +72,17 @@ def add_children(self, children: list[Entity] | list[PropertyGroup]):
         :param children: Add a list of entities as
             :obj:`~geoh5py.shared.entity.Entity.children`
         """
-        super().add_children(children)
-        self._property_groups = None
+        property_groups = self._property_groups or []
+
```

Comment:
Shouldn't we test the type of children? Ensure it's a data, object of PropertyGroup?
super() children was already doing that.

Reply:
Children cannot be "Objects"?